shopelectro/selenium/conditions.py

```diff
@@ -0,0 +1,24 @@
+from selenium.common.exceptions import StaleElementReferenceException
+
+
+def alphanumeric(text: str) -> str:
+    return ''.join(filter(lambda c: c.isalnum(), text))
+
+
+class AlnumPresentedInValue:
```

Comment:
@duker33 , 
`AlnumPresentedInValue` is an expected condition as [conditions](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/support/expected_conditions.py) from selenium. And it may instantiate an object.  

`WebDriverWait` can't use provided `SafeInput`, because it isn't a condition

`AlnumInValueCondition` sounds better?

Reply:
@artemiy312 , imho selenium conditions have not the best arch in the world =)
Condition interface implementing may contain some problems, but we can resolve them.
Let's leave you code and class name (choose one). I'll try propose some changes in separated PR